src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Random random = new Random();
+        return new Ball(ColorSupplier.getRandomColor(), random.nextInt(100));
```

Comment:
It's better to avoid creating a new instance of Random every time the method is called. Consider making 'random' a private instance variable of the Lottery class.